### PR TITLE
Workaround to increase compatibility with older systems

### PR DESCRIPTION
**Pull request by probonopd**
See https://github.com/darealshinji/AppImageKit-checkrt for details
https://github.com/aardappel/treesheets/issues/130#issuecomment-531594897

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -107,6 +107,10 @@ script:
       cp appdir/usr/bin/images/treesheets.svg appdir/usr/share/icons/hicolor/scalable/apps/ # FIXME
       ( cd appdir ; ln -s usr/bin/docs/ usr/bin/examples/ usr/bin/images/ usr/bin/scripts/ usr/bin/translations/ . )
       sed -i -e 's|1011|1002|g' appdir/usr/bin/treesheets # https://github.com/aardappel/treesheets/issues/130#issuecomment-528075693
+      # Workaround to increase compatibility with older systems; see https://github.com/darealshinji/AppImageKit-checkrt for details
```
Comment (TheAssassin) [inline .travis.yml]: Live'd be much easier with the linuxdeploy, where we have a plugin for this kind of stuff.

There's much custom weird undocumented script code in this file...

Comment (probonopd) [reply .travis.yml]: Possibly. But this I had available for copy-and-paste, and it is working for me.

**Comment by aardappel** [comment]
Hmm, hard for me to say wether this is a good idea, but I guess we'll merge for now..

**Comment by aardappel** [comment]
Don't suppose we can statically link it? :)

**Comment by TheAssassin** [comment]
You can, but as soon as you have any dependencies, you'd have to build those dependencies to statically link everything unavailable on older systems as well.

[appimagelint](https://github.com/TheAssassin/appimagelint) provides a reliable ABI-based compatibility check that can tell you whether you did that right (if you want to give it a try), it analyzes all binaries for their required libstdc++/libc versions and then compares that to what is known to be available on common Debian/Ubuntu releases.

**Pull request merged**